src/arrayReverse.test.js

```diff
@@ -8,13 +8,33 @@ describe(`Function 'arrayReverse':`, () => {
   });
 
   it(`should return an array`, () => {
+    const result = arrayReverse([]);
```

Comment:
The test case should check if the order of the strings is also reversed, not just the strings themselves. According to the task description, the function should reverse the strings and their order.